tools/sponge/preproc_scripts/fill_glorys_nn_monthly.csh

```diff
@@ -0,0 +1,39 @@
+#!/bin/tcsh
+#SBATCH --ntasks=1
+#SBATCH --job-name=fill_glorys_arctic
+#SBATCH --time=2880
+#SBATCH --partition=batch
+
+# Usage: sbatch fill_glorys_nn_monthly.sh <YEAR> <MONTH>
+# Original Author: Andrew Ross, modified by Theresa Morrison
+
+module load cdo
+module load nco/5.0.1
+module load gcp
+
+set year=$1
+set month=`printf "%02d" $2`
+
+set apath='/archive/Theresa.Morrison/datasets/glorys/GLOBAL_MULTIYEAR_PHY_001_030/monthly'
+
+# Regionally-slice and convert daily to monthly GLORYS reanalysis on archive beforehand.
+
+# dmget all of the files for this month from archive.
+dmget ${apath}/so/GLORYS_so_arctic_${year}_${month}.nc ${apath}/thetao/GLORYS_thetao_arctic_${year}_${month}.nc
```

Comment:
@andrew-c-ross  I think this combines the dmget. Nothing is being dmget in my testing, so I'm not sure if it is working.

Reply:
update: not working, it is writing over my files as expected.